api/src/opentrons/protocol_api/protocol_context.py

```diff
@@ -500,15 +506,21 @@ def load_module(
         by using :py:attr:`loaded_modules`.
 
         :param str module_name: The name or model of the module.
-        :param location: The location of the module. This is usually the
-                         name or number of the slot on the deck where you
-                         will be placing the module. Some modules, like
-                         the Thermocycler, are only valid in one deck
-                         location. You do not have to specify a location
-                         when loading a Thermocycler---it will always be
-                         in Slot 7.
+
+        :param location: The location of the module.
+
+            This is usually the name or number of the slot on the deck where you
+            will be placing the module, like ``1``, ``"1"``, or ``"D1"``. See :ref:`deck-slots`.
+
+            Some modules, like the Thermocycler, are only valid in one deck location.
+            You do not have to specify a location when loading a Thermocycler---it will always
+            be in Slot 7.
+
+            .. versionchanged:: 2.15
+                You can now specify a deck slot as a letter-number pair, like ``"D1"``.
```

Comment:
```suggestion
                You can now specify a deck slot with coordinates, like ``"D1"``.
```
And do we need a similar notice on `load_labware`?

Reply:
Yes, thank you.